nucliadb/src/nucliadb/search/search/chat/ask.py

```diff
@@ -222,7 +225,11 @@ async def _stream(self) -> AsyncGenerator[AskResponseItemType, None]:
         first_chunk_yielded = False
         with self.metrics.time("stream_predict_answer"):
             async for answer_chunk in self._stream_predict_answer_text():
-                yield AnswerAskResponseItem(text=answer_chunk)
+                if isinstance(answer_chunk, TextGenerativeResponse):
+                    yield AnswerAskResponseItem(text=answer_chunk.text)
+                elif isinstance(answer_chunk, ReasoningGenerativeResponse):
+                    yield ReasoningAskResponseItem(text=answer_chunk.text)
+
                 if not first_chunk_yielded:
```

Comment:
does the reasoning chunk count as "first chunk of the answer yelded"? to be confirmed with product.